### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by doxthree**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -29,14 +29,14 @@ You need credentials to use the service in your application. If you are interest
...
-|Functionality|Rust|Java|PHP|TypeScript|
-|---|---|---|---|---|
-|Check SHA256|&#9989;|&#9989;|&#9989;|&#9989;|
-|Check SHA256 list|&#9989;|&#10060;|&#10060;|&#9989;|
-|Check file|&#9989;|&#9989;|&#9989;|&#9989;|
-|Check file list|&#9989;|&#10060;|&#10060;|&#9989;|
+|Functionality|Rust|Java|PHP|TypeScript|C#|Python|
+|---|---|---|---|---|---|---|
+|Check SHA256|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|
+|Check SHA256 list|&#9989;|&#10060;|&#10060;|&#9989;|&#10060;|&#10060;|
```
Comment (unglaublicherdude) [inline Readme.md:37]:
```suggestion
|Check SHA256 list|&#9989;|&#10060;|&#10060;|&#9989;|&#9989;|&#10060;|
```

File: Readme.md [hunk 2]
```diff
@@ -29,14 +29,14 @@ You need credentials to use the service in your application. If you are interest
...
+|Check file|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|
+|Check file list|&#9989;|&#10060;|&#10060;|&#9989;|&#10060;|&#10060;|
```
Comment (unglaublicherdude) [inline Readme.md:39]:
```suggestion
|Check file list|&#9989;|&#10060;|&#10060;|&#9989;|&#9989;|&#10060;|
```

File: Readme.md (hunk 1 above)
Comment (unglaublicherdude) [reply Readme.md:37]: c# does support list requests

File: Readme.md (hunk 2 above)
Comment (unglaublicherdude) [reply Readme.md:39]: c# does support list requests

**Review by secana — APPROVED** [review]

**Review by secana — APPROVED** [review]

**Comment by unglaublicherdude** [comment]
Not to sure, but maybe also call it dotnet instead of C# as the folder already follows this name?

**Comment by secana** [comment]
good point. The library isn't C# only, F# and VB works fine as well

**Review by sfotiadis — APPROVED** [review]

**Review by unglaublicherdude — APPROVED** [review]

**Pull request merged**